### PR TITLE
feat: add new command menu to studio

### PR DESCRIPTION
**Pull request by charislam**
Add new command menu to studio. Ports over commands from old menu (with some minor reorganization), then adds:

- Branch switcher
- Project switcher

### CODE REVIEW

File: apps/studio/components/interfaces/App/CommandMenu/Support.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { COMMAND_MENU_SECTIONS } from './CommandMenu.utils'
+
+const useSupportCommands = () => {
+  const commands = useMemo(
+    () =>
+      [
+        {
+          id: 'support',
+          name: 'Support',
+          route: '/support',
```
Comment (saltcod) [inline apps/studio/components/interfaces/App/CommandMenu/Support.tsx:16]: this gives me a 404 — prob just on staging? 
https://studio-staging-git-feat-new-command-menu-studio-supabase.vercel.app/dashboard/support

File: apps/studio/components/layouts/AdvisorsLayout/Advisors.Commands.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { useParams } from 'common'
```
Comment (saltcod) [inline apps/studio/components/layouts/AdvisorsLayout/Advisors.Commands.tsx:1]: can remove this guy now?

**Review by saltcod — APPROVED** [review]

File: apps/studio/components/layouts/AdvisorsLayout/Advisors.Commands.tsx (hunk 2 above)
Comment (charislam) [reply apps/studio/components/layouts/AdvisorsLayout/Advisors.Commands.tsx:1]: I think we should keep right? This is just the navigation item (I removed the one that shows the lints)

File: apps/studio/components/interfaces/App/CommandMenu/Support.tsx (hunk 1 above)
Comment (charislam) [reply apps/studio/components/interfaces/App/CommandMenu/Support.tsx:16]: Ahhh, no, good catch, this is under www not studio so needs to be a full route

**Pull request merged**